wdio.conf.js

```diff
@@ -0,0 +1,395 @@
+"use strict";
+
+const commonjs = require("vite-plugin-commonjs");
+
+exports.config = {
+
+    /*
+     *
+     * ====================
+     * Runner Configuration
+     * ====================
+     * WebdriverIO supports running e2e tests as well as unit and component tests.
+     */
+    runner: ["browser", {
+        viteConfig: {
+            resolve: {
+                alias: {
+                    util: "rollup-plugin-node-polyfills/polyfills/util",
+                    path: "rollup-plugin-node-polyfills/polyfills/path",
+                    assert: "rollup-plugin-node-polyfills/polyfills/assert"
+                }
+            },
```

Comment:
not sure this is needed.  IMHO, the webpack bundled file should have included node.js built-in modules, as configured: https://github.com/eslint/eslint/blob/8bcbf11b6050418262ffa8e0ca37f365ae92e7ce/webpack.config.js#L47

Reply:
WebdriverIOs component testing framework uses Vite under the hood, which allows Rollup plugins.

Reply:
well, I see!

the plugin was applied to `/lib/linter/linter.js`, but not `tests/lib/linter/linter.js`. 😄